src/Game.java

```diff
@@ -0,0 +1,115 @@
+import java.util.*;
+
+public class Game {
+    private final static int PLAY = 1;
+    private final static int EXIT = 2;
+    private static final int MAX_ERRORS = 8;
+
+    private Scanner scanner = new Scanner(System.in);
+
+    private String wordToFind;
+    private char[] guessedWord;
+    private int mistakesMade;
+    private ArrayList<String> words = new ArrayList<>();
+
+    Keywords keywords = new Keywords();
+    WordsFromFile wordsFromFile = new WordsFromFile();
+
+    public void hangman() {
+        int option = -1;
+        while (option != EXIT) {
+
+            printOption();
+
+            option = scanner.nextInt();
+            scanner.nextLine();
+
+            switch (option) {
+                case EXIT:
+                    System.out.println("Bye bye!");
+                    scanner.close();
+                    break;
+                case PLAY:
+                    play();
+                    break;
+                default:
+                    System.out.println("Wybrałeś nieprawidłową opcję");
+            }
+        }
+    }
+
+    private void printOption(){
+        System.out.println("Wybierz opcję:");
+        System.out.println("1 - graj");
+        System.out.println("2 - koniec programu");
+    }
+
+    private void newGame() {
+        mistakesMade = 0;
+        words.clear();
```

Comment:
Why are clearing words array?

Reply:
to make sure it is empty and there will be no error for me

Reply:
Isn't this an array storing words that game is randomly choosing from?

Reply:
No, such an array is in the Keywords class.

Reply:
So what is this array storing?  What words is it storing?

Reply:
Do you understand my point? We have been talking about it for couple weeks now. This field name is bad because it can mean pretty much anything. What words is it storing? Or is it storing letters?